.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm guessing tht you meant to add this folder of editor settings, since this is for your working environment. 

For other repos, vscode, terraform, version managers, will all make working directories that you don't want  to commit. 
You can use a .gitignore file to remove those files from git status & diff + prevent them from being committed ❌